nvflare/apis/workspace.py

```diff
@@ -164,8 +164,8 @@ def get_job_meta_path(self, job_id: str) -> str:
     def get_site_privacy_file_path(self):
         return self.get_file_path_in_site_config(WorkspaceConstants.PRIVACY_CONFIG)
 
-    def get_client_custom_dir(self) -> str:
-        return os.path.join(self.get_site_config_dir(), WorkspaceConstants.CUSTOM_FOLDER_NAME)
+    def get_client_custom_dir(self, job_id: str) -> str:
```

Comment:
No, this is incorrect!
get_client_custom_dir returns the path to the "custom" folder in "local".
To get the path to job's custom folder, you use get_app_custom_dir.

Please read the comment at the beginning of the workspace.py for the structure of the workspace.

Reply:
I missed up the get_client_custom_dir() and get_app_custom_dir(). Changed back.